src/sql/workbench/contrib/notebook/browser/cellViews/code.component.html

```diff
@@ -13,5 +13,8 @@
 			</div>
 		</div>
 		<collapse-component *ngIf="cellModel.cellType === 'code' && cellModel.source && cellModel.source.length > 1" [cellModel]="cellModel" [activeCellId]="activeCellId"></collapse-component>
+		<div #parameter class="parameter" *ngIf="cellModel.cellType === 'code' && cellModel.isParameter">
+			<span>Parameters</span>
```

Comment:
We don't keep text in templates. Please add this text as a localized string to the component file, then pass in the variable here.